sdk/core/Azure.Core/tests/TestClients/TestResourceOperations.cs

```diff
@@ -6,6 +6,7 @@
 using System.Threading.Tasks;
 using Azure.Core.Pipeline;
 using Azure.Core.TestFramework;
+using Azure.ResourceManager;
 using Azure.ResourceManager.Core;
```

Comment:
`using Azure.ResourceManager.Core;` can be removed, right?

Reply:
Not we are keeping the namespace for the base types like OperationBase which is referenced here